src/gui/tray/ActivityData.h

```diff
@@ -71,6 +71,7 @@ class Activity
     QString _file;
     QUrl _link;
     QDateTime _dateTime;
+    qint64 _expireAtMsecs = -1;
```

Comment:
why letting errors expire ?
user may just be somewhere else and will miss them even though they are important

Reply:
> why letting errors expire ?
> user may just be somewhere else and will miss them even though they are important

@mgallien 

Otherwise, a user will get confused because the folder is wiped already but the message is always visible unless the user restarts. For some users, restart rarely happens, they may not turn off their device, but, rather put it to sleep. Then, errors would be present even after days of not restarting.

When renaming a file/folder a user will likely open the tray to see what's happened (why did the folded got wiped), so chances are errors will be seen. But, if they are not seen minutes after rename has failed, does it even make sense to keep them there for hours/days? I can add a call to display those errors with OS notification. So, a popup will get displayed along with the error. Will this be better/make sense?

Reply:
yes, rename that will result in the directory being wiped out due to the rename conflict should properly be notified to the user
but keep your existing logic to also have it inside our stuff